igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildCache.groovy

```diff
@@ -0,0 +1,83 @@
+package com.netflix.spinnaker.igor.build
+
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import redis.clients.jedis.Jedis
+import redis.clients.jedis.JedisPool
+
+
+/**
+ * Shared cache of build details
+ */
+@SuppressWarnings(['PropertyName', 'DuplicateNumberLiteral'])
+@Service
+@Slf4j
+class BuildCache {
```

Comment:
would replace this with something id based like the dockerregistrycache - https://github.com/spinnaker/igor/blob/f0b58682daeff0c626eb4aaf3da033786fa19f8e/igor-web/src/main/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCache.groovy